src/emcee.jl

```diff
@@ -0,0 +1,226 @@
+struct Ensemble{D} <: MHSampler
+    n_walkers::Int
+    proposal::D
+end
+
+struct Walker{T<:Union{Vector,Real,NamedTuple},L<:Real}
+    params::T
+    lp::L
+end
+
+Walker(t::Transition) = Walker(t.params, t.lp)
+Walker(model::DensityModel, params) = Walker(params, logdensity(model, params))
```

Comment:
That's the same as `Transition` it seems, so one could just use `Transition` instead it seems?

Reply:
I still think one could just use `Transition` instead of a separate `Walker`. If the name is confusing in this context, one could just define
```julia
const Walker = Transition
```